apps/web/components/question/Question.test.tsx

```diff
@@ -94,7 +103,7 @@ describe("Question", () => {
   // check question with code reference
   it("QuestionMultiChoice: renders with code reference", () => {
     // get data
-    const walkthroughData = useWalkthroughTestData();
+    const walkthroughData = useWalkthroughTestData999();
```

Comment:
As we prepare for significant uptick in use cases do we feel this approach is scalable?